### DISCUSSION TITLE
How Can I Get Started

### DISCUSSION THREAD

**Discussion by himanshu1221** [wagtail/gsoc#4] [Q&A] [answered]
Respected Sir/madam 
I am Himanshu chhatwal a second-year undergraduate I have just entered my second year and I am new to open source I am well aware of JavaScript, and nodeJs. I went thru some ideas and found one of my interests i.e "Dark Theme For Admin Interface" my query was how can get started and how can I connect to mentors for the same idea 

Hope to  hear from you soon

Regards 
Himanshu

**Comment by SAMAD101** [comment]
I'm new to Wagtail as well. I asked on Slack, and they said to wait for mentors to address us, they'll reach out to us in about a week. Till then, you can contribute to existing repositories of [Wagtail](https://github.com/wagtail).

**Reply by himanshu1221** [reply]
> I'm new to Wagtail as well. I asked on Slack, and they said to wait for mentors to address us, they'll reach out to us in about a week. Till then, you can contribute to existing repositories of [Wagtail](https://github.com/wagtail).

Thanks For Answering!

**Reply by VINAYSH778** [reply]
Plzz bro tell me how to apply a letter to joining a contribut the project

**Reply by sahil-sagwekar2652** [reply]
> > I'm new to Wagtail as well. I asked on Slack, and they said to wait for mentors to address us, they'll reach out to us in about a week. Till then, you can contribute to existing repositories of [Wagtail](https://github.com/wagtail).
> 
> Thanks For Answering!

You can find more info about setting up the codebase for development [here](https://docs.wagtail.org/en/stable/contributing/index.html)

**Comment by thibaudcolas** [comment, marked as answer]
Welcome @himanshu1221! Please read through our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md) which contains all the "getting started" steps we expect from contributors.

**Comment by HarshithaPerumalla** [comment]
Hi Everyone,
I am Harshitha Perumalla, a final year computer science  undergrad. I am new to open source contributions and exploring open source. But I have strong skills in  python, javascript and Django. I would love to contribute to your organization but could you please tell me how to get started?

**Reply by thibaudcolas** [reply]
Hi @HarshithaPerumalla, this project is specifically for GSoC participants. I’d suggest you look at other resources that would be more relevant to your situation.

**Comment by praptisharma28** [comment]
Hello all, I am  Prapti Sharma, a passionate Django Developer, I know HTML, CSS, JS, React, Django and familiar with python, C, C++. I want to contribute in this organisation. Guide me if you can. Tell me if there is any specific task for me to do. I will be very happy to contribute, Thankyou for your time if you read this.

**Reply by thibaudcolas** [reply]
https://github.com/wagtail/gsoc#2024

**Comment by iqraakhtar09** [comment]
Hello everyone,
I'm iqra I'm new here. I'm highly excited and full of potential to work with Wagtail. I have experience in Django, Flask, and ReactJS. My main focus is on leetcode. I love to solve different problems, understand those problems, and find a new solution. I'm also a mentor at lablab.ai. I participated in different hackathons to enhance my skills. I hope i will learn a lot from this community. I want to know is this community have any slack where i can discuss problems and other things. @thibaudcolas 
Thank you

**Comment by LQNguyen05-max** [comment]
Hi everyone. 

My name is Leo, and I have a passion working with Wagtail. I have experience working with Django, Python and JavaScript. Today, I want to contribute to the organization for the GSOC 2026. Can anyone guide me on how to start?
